examples/with-mdx/src/pages/index.mdx

```diff
@@ -15,3 +15,15 @@ Written by: {new Intl.ListFormat('en').format(authors.map(d => d.name))}.
 Published on: {new Intl.DateTimeFormat('en', {dateStyle: 'long'}).format(published)}.
 
 <Counter client:idle>This is a **counter**!</Counter>
+
+## Syntax highlighting
+
+We also support syntax highlighting in MDX out-of-the-box! This example uses our default [Shiki theme](https://github.com/shikijs/shiki). See full configuration options in [our markdown documentation](https://docs.astro.build/en/guides/markdown-content/#syntax-highlighting).
```

Comment:
Does `shiki-twoslash` have docs for this? I'd rather point to the fork we're actually using than the original.

Reply:
I'll just link to the integration README!